theano/tensor/basic.py

```diff
@@ -4447,6 +4447,11 @@ class Flatten(Op):
     __props__ = ("outdim",)
 
     def __init__(self, outdim=1):
+        warnings.warn(
```

Comment:
Yes, in addition. See comment for GpuFlatten.

The docstring is above at line 4437.

```python
class Flatten(Op):
     """
     Flatten a tensor.
 
     Flattens a tensor to `outdim` dimensions by preserving the leading
     outdim - 1 shape components.
 
     """
```